sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java

```diff
@@ -63,6 +71,10 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+// This class is executed synchronously as it runs into a NullPointerException when attempting to dispose
+// connectionProcessor in parallel runs.
+@Execution(ExecutionMode.SAME_THREAD)
+@Isolated
```

Comment:
Are both these required? Can just `@Isolated` be used here?

Reply:
Isolated is different than Execution. Isolated indicates to JUnit to run this test class in a sequential mode. Execution indicates to run the tests in either single threaded or multi threaded mode.

Reply:
Yeah, I was expecting this test to run in multi-threaded mode and just had to be isolated from running it with other tests.